src/components/Header/header.less

```diff
@@ -65,6 +65,12 @@
   }
 
   &__account {
+    vertical-align: middle;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: center;
+    align-items: center;
+
```

Comment:
Just one note, if there is no message about the incorrect network, does that css won't place the account info in the middle of the space after the logo/menu?